docs/LV_client/tutorials/wheel_integration.md

```diff
@@ -0,0 +1,31 @@
+# Wheel integration
+
+## Pre-requisites
+1. The **mono_closed_loop_example_wheel.vi** and the **mono_replay_example_wheel.vi** use the **G920 for Xbox One and PC**, more information [here](https://www.logitechg.com/en-us/products/driving/driving-force-racing-wheel.html).
+1. Install the Logitech G HUB to obtain the latest drivers for the wheel from [here](https://www.logitechg.com/en-us/innovation/g-hub.html)
```

Comment:
Maybe try: Ensure you have the latest drivers for the wheel from [Logitech G HUB](https://www.logitechg.com/en-us/innovation/g-hub.html)